lib/galaxy/managers/notification.py

```diff
@@ -0,0 +1,61 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional
+)
+
+from sqlalchemy import false
+from galaxy import model
+from galaxy.exceptions import (
+    ObjectNotFound,
+)
+from galaxy.model.scoped_session import galaxy_scoped_session
+
+class NotificationManager:
+    """Interface/service object shared by controllers for interacting with notifications."""
+
+    def __init__(self, sa_session: galaxy_scoped_session):
+        self.sa_session = galaxy_scoped_session
```

Comment:
```suggestion
        self.sa_session = sa_session
```
this is why your tests are failing